dts/bindings/timer/nordic,nrf-timer.yaml

```diff
@@ -34,4 +34,10 @@ properties:
       category: required
       description: Human readable string describing the device (used by Zephyr for API name)
       generation: define
+
+    prescaler:
+      type: int
+      category: required
+      description: Prescaler value denotes number of TIMER ticks in 1 second
```

Comment:
I don't think this description is correct.

Reply:
Yeah, isn't it fq = 16 Mhz / 2^prescaler?

Reply:
fixed